### ISSUE TITLE
Change output location

### ISSUE THREAD

**Issue by coreybruce** [enigma9o7/smlinux#12]
Hi just thought it might be a better idea to make the output location for the build is inside the smlinux folder instead of dumping it in the users home folder

**Comment by enigma9o7** [comment]
> Hi just thought it might be a better idea to make the output location for the build is inside the smlinux folder instead of dumping it in the users home folder, I assume it's supposed to make a build folder inside smlinux and this is a bug

For the "smlinux" folder I use ~/.local/share/sm64pc for historical reasons; the was the folder the original fork stored its config file and savegame, so I used that for smlinux config file, and the actual smlinux script is installed in the first directory in your path, and I'm not going to change all at this point.  And both of those would certainly not be a convenient home folder.

However, I see your point, maybe some people don't want to keep stuff in home for whatever reason.  So I could add an entry to config file to define home folder, by default it'll be $HOME but user could change it if they wish, how would that work?   Wouldn't be too hard to implement, if good enuff let me know and I'll add it.

**Comment by coreybruce** [comment]
yeah for the save game that's good and fine but for building it should all self contain itself in the project folder, I am currently looking through the code trying to find where it creates the bin and sm64ex at the moment, if I can just make it all stay self contained that would be much nicer for everyone. :) 

I think it would just be better and a lot cleaner to to make it create a build folder inside smlinux with the bin and sm64ex folder inside that, there isn't any need to have it dump it all in a users home folder.
